pkg/infoschema/infoschema_test.go

```diff
@@ -850,3 +851,12 @@ func TestInfoSchemaCreateTableLike(t *testing.T) {
 	require.Equal(t, tblInfo.Indices[0].Name.O, "idx")
 	require.Equal(t, tblInfo.Indices[0].ID, int64(1))
 }
+
+func TestCheckDefaultValueForTidbTxnMode(t *testing.T) {
```

Comment:
No need this test, already covered by https://github.com/pingcap/tidb/blob/af44de3abbb25dd1c9864c6dc261a6763e7f7626/tests/integrationtest/t/infoschema/tables.test#L178

Reply:
And not sure does this change will affects upgrade from v3.x.

Reply:
@Defined2014 
Thanks for your review!
I updated test!
fix commit: https://github.com/pingcap/tidb/pull/48500/commits/033b8f9480b869a3a73bb19dd9848d2c7c202466